dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/Launcher.java

```diff
@@ -72,11 +72,21 @@ public int createPlatform(String[] args) {
         initProps.put("org.apache.aries.blueprint.preemptiveShutdown", Boolean.toString(false));
 
         BootstrapConfig bootProps = createBootstrapConfig();
+        List<String> newArgs = new ArrayList<String>();
+
+        if (args != null && args.length > 0) {
+            newArgs.addAll(Arrays.asList(args));
+        }
+
+        // If we are running on Java 18+, then we need to explicitly enable the security manager
+        if (javaVersion() >= 18) {
+            newArgs.add("-Djava.security.manager=allow");
```

Comment:
I expected this to be a System.setProperty, not adding it to the arguments.  I believe there are the program arguments, not the Java arguments.

Reply:
It needs to be before createBootstrapConfig since that method does a doPriv in it and it will blow up if the property is not set.

Reply:
Done